### DISCUSSION TITLE
Regarding setting scaling_factor=0.18215 instead of 0.13025 in stage 1 vae training

### DISCUSSION THREAD

**Discussion by MAFLIXD** [danny-avila/LibreChat#3121] [Troubleshooting]
### What happened?

https://github.com/hpcaitech/Open-Sora/issues/493

### Steps to Reproduce

https://github.com/hpcaitech/Open-Sora/issues/493

### Relevant log output

```shell
regarding setting scaling_factor=0.18215 instead of 0.13025 in stage 1 vae training
```

### Screenshots

https://github.com/hpcaitech/Open-Sora/issues/493

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Need a little more context here
